test/scaladoc/run/t8755c/ploogin_1.scala

```diff
@@ -0,0 +1,31 @@
+
+package t8755
+
+import scala.tools.nsc.{Global, Phase}
+import scala.tools.nsc.plugins.{Plugin, PluginComponent}
+import scala.reflect.io.Path
+import scala.reflect.io.File
+
+/** A test plugin.  */
+class Ploogin(val global: Global) extends Plugin {
+  import global._
+
+  val name = "ploogin"
+  val description = "A sample plugin for testing."
+  val components = List[PluginComponent](TestComponent)
+
+  private object TestComponent extends PluginComponent {
+    val global: Ploogin.this.global.type = Ploogin.this.global
+    override val runsBefore = Nil
+    val runsAfter = List("refchecks")
```

Comment:
other way around. So many combinations. Where is Scalacheck to fuzz your test source when you need it?